src/python/pants/backend/python/goals/setup_py_test.py

```diff
@@ -227,6 +227,129 @@ def test_generate_chroot(chroot_rule_runner: RuleRunner) -> None:
     )
 
 
+def test_generate_chroot_entry_points(chroot_rule_runner: RuleRunner) -> None:
```

Comment:
Copied the other chroot test so this one could be trimmed down to only test the entry points stuff. I like tests that doesn't test too much, but if you feel like keeping it all in a single test, I'm ok with that too.. ;)

Reply:
Yeah, if you're willing, trimming down to just setup entrypoints would be great. Certainly the .js and .pyi treatment is not needed in this test.